src/pages/BookPage/bookPage.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import './style.scss';
+import { MDBContainer, MDBRow, MDBCol, MDBBtn, MDBIcon } from 'mdbreact';
+import { useTranslation } from 'react-i18next';
+const BookPage = () => {
+  const { t } = useTranslation();
+
+  const bookPage = {
```

Comment:
This variable name is little bit confusing  
try to make it `bookObject` or `bookData` or just `book` or anything else